src/popup/router/components/Modals/AccountDetails.vue

```diff
@@ -73,6 +73,30 @@
             :display-filter="showSearchBox"
           />
         </div>
+        <div
+          v-if="activeTab === 'names'"
+          class="names-list-wrapper"
+        >
+          <div class="tabs">
```

Comment:
This styling is something more common so I suggest to create separate component for the tabs.